machine.go

```diff
@@ -85,6 +85,7 @@ const (
 	MachineTypeMicrovm string = "microvm"
 	MachineTypePC35    string = "q35"
 	MachineTypePC      string = "pc"
+	MachineTypeVirt	   string = "virt"
```

Comment:
This Machine Type is aarch64 only, on x86_64; this isn't available.

I think we want to move this into machine_aarch64.go, and include the `// +build aarch64` at the top

I think this should keep the value out of the x86_64 qcli path so folks don't accidentally use it.